status-go-version.json

```diff
@@ -2,7 +2,7 @@
     "_comment": "DO NOT EDIT THIS FILE BY HAND. USE 'scripts/update-status-go.sh <tag>' instead",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.34.0-beta.8",
-    "commit-sha1": "9d7c570593b1f88e9688204d8e1041d57b98da1f",
-    "src-sha256": "1kwxf0h80vdj493c7s3lpmdjpbc72plbll0rj1vv7kzf8a84ff2k"
+    "version": "develop",
+    "commit-sha1": "464c30a43019fe9b774036f685f99c54fc22b882",
+    "src-sha256": "0p6qryhszs48qxkb3h2j1p0wip3ii9928r34yi0z1vfakwk4056f"
```

Comment:
Please use the `v0.35.1` tag instead of `develop`.

Reply:
you don't need to update the version file is status-go?

Reply:
Well, technically, yes, in practice it has no effect, but you're right I should do it.
Try now, I updated the tag `v0.35.1` tag, it should point at https://github.com/status-im/status-go/commit/4769fb91c3247c72c21013bf36194843d8d72d18.